cmake/ExternalLibs.cmake

```diff
@@ -0,0 +1,27 @@
+# RMM
+if (USE_RMM)
+  # Use Conda env if available
+  if(DEFINED ENV{CONDA_PREFIX})
```

Comment:
I don't think it's a good idea to check for conda.  During packaging people should create isolated build environment, either by manipulating the environment variable, or using cgroup.

Reply:
This is commonly done in Rapids packages. The alternative is to manually specify location of RMM libraries and headers.

Reply:
I suppose you can specify CMAKE_PREFIX_PATH to CONDA_PREFIX. Let me remove mention of Conda.

Reply:
Does it export a cmake config that can be used by find package?

Reply:
It does not.

Reply:
@hcho3 Opened an issue in https://github.com/rapidsai/rmm/issues/455,  I can work on it if the feature request is accepted.  cc @jrhemstad

Reply:
Removed check for `CONDA_PREFIX`.